tensorflow/compiler/xla/service/gpu/ir_emitter.cc

```diff
@@ -23,12 +23,8 @@ limitations under the License.
 #include "llvm/IR/DerivedTypes.h"
 #include "llvm/IR/Instructions.h"
 #include "llvm/IR/Module.h"
-<<<<<<< HEAD
-#include "llvm/IR/ModRef.h"
```

Comment:
this header is needed for the new argMemOnly argument added in https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/commit/ca8fa12fda3c2149399a5b3dcb164c6999351b48

Without it, this will likely fail to build.

Reply:
> this header is needed for the new argMemOnly argument added in [ca8fa12](https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/commit/ca8fa12fda3c2149399a5b3dcb164c6999351b48)
> 
> Without it, this will likely fail to build.

Yes, I added it by cherry-pick https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/commit/ca8fa12fda3c2149399a5b3dcb164c6999351b48, forgot to add it to my commit...it's added now.